pkg/cmd/pr/checks/checks.go

```diff
@@ -61,12 +61,19 @@ func NewCmdChecks(f *cmdutil.Factory, runF func(*ChecksOptions) error) *cobra.Co
 	cmd := &cobra.Command{
 		Use:   "checks [<number> | <url> | <branch>]",
 		Short: "Show CI status for a single pull request",
-		Long: heredoc.Doc(`
+		Long: heredoc.Docf(`
 			Show CI status for a single pull request.
 
 			Without an argument, the pull request that belongs to the current branch
 			is selected.
-		`),
+
+			The %[1]s--json%[1]s flag outputs the check information in JSON format. It includes:
+
+			- %[1]sbucket%[1]s: Categorizes checks as %[1]spass%[1]s, %[1]sfail%[1]s, %[1]spending%[1]s, %[1]sskipping%[1]s, or %[1]scancel%[1]s.
+			- %[1]sstate%[1]s: Provides the specific status like %[1]sSUCCESS%[1]s, %[1]sFAILURE%[1]s, %[1]sPENDING%[1]s, etc.
+
+			The %[1]sbucket%[1]s field simplifies statuses into categories, while %[1]sstate%[1]s gives the exact status.
```

Comment:
See my bigger comment. I didn't like how the output looked, here, so I'm leaning towards something much simpler:

```suggestion
			When the `--json` flag is used, it includes a `bucket` field, which categorizes the `state`
field into  `pass`, `fail`, `pending`, `skipping`, or `cancel`.
```